### DISCUSSION TITLE
Changing Min Max Servo timings (solved)

### DISCUSSION THREAD

**Discussion by merely-present** [ArminJo/ServoEasing#37] [Q&A] [answered]
Firstly, thanks for the free software.

I am not really sure how to change Min Max servo timings with this library. Looking at the source it seems it is a private hard coded value? Am I missing something? I haven't found anything about it in the examples as of yet.

Just as I finally decided to make this post I figured it out. Its with an attach overload

**Comment by ArminJo** [comment, marked as answer]
You are right, it is hand coded. The parameters have to be inserted in 
attach(int aPin, int aInitialDegree, int aMicrosecondsForServo0Degree, int aMicrosecondsForServo180Degree)
**The values can be determined by the EndPositionsTest example.**
